pkg/environments/environment.go

```diff
@@ -22,7 +22,7 @@ const (
 	IntegrationEnv string = "integration"
 
 	EnvironmentStringKey string = "OCM_ENV"
-	EnvironmentDefault          = DevelopmentEnv
+	EnvironmentDefault          = IntegrationEnv
```

Comment:
Why this change?

Reply:
I will remove it. I am adding it when debugging integration tests locally